docs/configuration-reference/components/dex.md

```diff
@@ -138,8 +138,8 @@ google_client_id     = "1234567890123-SqDIX1KFvKPYmuV9Sa8eL92cvxtS3TuP.apps.goog
 google_client_secret = "63zYPITtigLxLaYBEjNP9Taw"
 
 # A random secret key (create one with `openssl rand -base64 32`)
-dex_static_client_gangway_secret = "2KBvQkjOZdc3iHt4KSb9GUECdenH/VDl04TwMdSyPcs="
-dex_static_client_gangway_id     = "gangway"
+dex_static_client_clusterauth_secret = "2KBvQkjOZdc3iHt4KSb9GUECdenH/VDl04TwMdSyPcs="
+dex_static_client_clusterauth_id     = "cluster-auth"
```

Comment:
Why variable name is `clusterauth`, but value is `cluster-auth`? Can we be consistent here?

Reply:
I wanted to use `cluster-auth` but I think `dex_static_client_cluster-auth_id` looks weird and `dex_static_client_cluster_auth_id` doesn't look quite right. I can change everything to `clusterauth` though, does that sound ok?